api/wallet/spec/definitions/SourceID.yaml

```
@@ -1,3 +1,4 @@
-description: Идентификатор источника денежных средств
+---
+description: Cash source identifier
```

Может Funds, как выше использовали?
```suggestion
description: Funds source identifier
```